business/workloads.go

```diff
@@ -7,6 +7,7 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+	"reflect"
```

Comment:
Generally using `reflect` is known to be a performance hit and considering we're parsing logs here it could have an impact. Is there another way to accomplish the same thing without reflect?